src/chart/bar/BarView.js

```diff
@@ -563,3 +647,61 @@ function setLargeStyle(el, seriesModel, data) {
     el.style.lineWidth = data.getLayout('barWidth');
 }
 
+function setLargeBackgroundStyle(el, backgroundModel, data) {
+    var borderColor = backgroundModel.get('borderColor') || backgroundModel.get('color');
+    var itemStyle = backgroundModel.getItemStyle(['color', 'borderColor']);
+
+    el.useStyle(itemStyle);
+    el.style.fill = null;
+    el.style.stroke = borderColor;
+    el.style.lineWidth = data.getLayout('barWidth');
+}
+
+function getBackgroundShape(isPolar, isHorizontalOrRadial, layout, coordLayout) {
```

Comment:
(I am not sure)
By convention a name getXxx always means a "getter" which returns something existing rather than create something.
Probably createXxx is better here.